.gitmodules

```diff
@@ -41,3 +41,9 @@
         path = korrel8r
         url = https://github.com/korrel8r/korrel8r.git
         branch = v0.7
+[submodule "perses-operator"]
+	path = perses-operator
+	url = https://github.com/perses/perses-operator.git
+	#branch = TBD release brach 
+	# sub-module should point to tag v0.1.0
+	ignore = all #ignore all `git submodule updates` to this submodule (will be manually updated to accomdate usage of tags)
```

Comment:
manually updating might be cumbersome. We can create a release branch from the operator side

Reply:
I updated this to point at the release branch I created in /perses/perses-operator and will update the .gitmoduels to  `branch=release-v0.1.0`. 

[https://github.com/perses/perses-operator/tree/release-v0.1.0](https://github.com/perses/perses-operator/tree/release-v0.1.0) it points to the same commit as the tag v0.1.0 ([e56893154adfc65e667efeb3a3a112042f1b28df](https://github.com/perses/perses-operator/commit/e56893154adfc65e667efeb3a3a112042f1b28df))